### PR TITLE
New testcase for pktgen testing

### PR DESCRIPTION
**Pull request by svelamal**
### Description of PR

  Testcase does the following steps:
    1. Check max CPU utilized , number of core and dump files before starting the run
    2. Configure pktgen traffic and start it, check if the pktgen process has generated the packets and 
       if the packets are seen across the interface
    3. Check max CPU utilized , number of core and dump files after the run is complete and verify if there any additional 
       core/dump files

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ x] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ x] 202205

#### How did you verify/test it?
Ran it on Cisco M64 devices (T0 and dualtor topo) and it passed

### CODE REVIEW

**Review by wsycqyz — APPROVED** [review]
LGTM

**Review by XuChen-MSFT — APPROVED** [review]
looks good to me

**Comment by wsycqyz** [comment]
@svelamal Please help raise another PR to fix the syntax error in this testcase:

-------------------------------- live log call ---------------------------------
13:03:35 __init__.pytest_runtest_call             L0040 ERROR  | Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/_pytest/python.py", line 1788, in runtest
    self.ihook.pytest_pyfunc_call(pyfuncitem=self)
  File "/usr/local/lib/python3.8/dist-packages/pluggy/_hooks.py", line 493, in __call__
    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
  File "/usr/local/lib/python3.8/dist-packages/pluggy/_manager.py", line 115, in _hookexec
    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
  File "/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py", line 113, in _multicall
    raise exception.with_traceback(exception.__traceback__)
  File "/usr/local/lib/python3.8/dist-packages/pluggy/_callers.py", line 77, in _multicall
    res = hook_impl.function(*args)
  File "/usr/local/lib/python3.8/dist-packages/_pytest/python.py", line 194, in pytest_pyfunc_call
    result = testfunction(**testargs)
  File "/var/src/sonic-mgmt_vms20-t0-7050cx3-2_646f1402735219c3e544409d/tests/test_pktgen.py", line 87, in test_pktgen
    pytest_assert(cpu_before > cpu_threshold, "Cpu util was above threshold {} for atleast 1 process \
TypeError: '>' not supported between instances of 'list' and 'int'

FAILED

File: tests/test_pktgen.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import pytest
+import random
+import re
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any'),
+    pytest.mark.device_type('vs')
```
Comment (XuChen-MSFT) [inline tests/test_pktgen.py:11]: HI @svelamal 

Is it necessary run this test on vs?  Or just "pytest.mark.asic('cisco')" is enough ?

+ @yutongzhang-microsoft  for viz

**Pull request merged**